buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -107,7 +108,7 @@ Requires: python3-PyYAML
 Requires: python3-mock
 %{?fedora:Requires: python3-psutil}
 %{?fedora:Requires: python3-requests}
-Requires: boost-python3-devel
+Requires: boost169-python3-devel
```

Comment:
This needs to be in the `%if 0%{el7}` section below. Also please add `python36-dateutil`

Reply:
Thanks. One day I will get the hand of these files...